ビルドエラー

@hi-kondo 
``seed_r7_robot_interface``をビルドすると下記エラーが出ます。
```
Starting  >>> seed_r7_robot_interface                                          
_______________________________________________________________________________
Errors     << seed_r7_robot_interface:cmake /home/seed/ros/kinetic/logs/seed_r7_robot_interface/build.cmake.001.log
CMake Error at /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:196 (message):
  catkin_package() the catkin package 'nav_msgs' has been find_package()-ed
  but is not listed as a build dependency in the package.xml
Call Stack (most recent call first):
  /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:102 (_catkin_package)
  CMakeLists.txt:30 (catkin_package)
```
package.xmlにdependタグが無いと怒られているようです。
ただし、PC環境によっては怒られない時もあります。
CMakeList.txtとpackage.xmlのどちらを修正するのが正しいでしょうか？

--- a/seed_r7_robot_interface/CMakeLists.txt
+++ b/seed_r7_robot_interface/CMakeLists.txt
@@ -5,9 +5,7 @@ project(seed_r7_robot_interface)
 
 # catkin ROS
 find_package(catkin REQUIRED COMPONENTS
-  rospy roscpp tf std_msgs sensor_msgs roslib
-  trajectory_msgs geometry_msgs 
-  message_generation
+  rospy roscpp tf roslib
   seed_r7_description
   seed_smartactuator_sdk
 )
@@ -29,8 +27,7 @@ endif()
 
 catkin_package(
   CATKIN_DEPENDS
-  roscpp tf std_msgs sensor_msgs roslib
-  trajectory_msgs geometry_msgs 
+  roscpp tf roslib
   seed_r7_description
   DEPENDS
   INCLUDE_DIRS include

--- a/seed_r7_robot_interface/CMakeLists.txt
+++ b/seed_r7_robot_interface/CMakeLists.txt
@@ -6,7 +6,7 @@ project(seed_r7_robot_interface)
 # catkin ROS
 find_package(catkin REQUIRED COMPONENTS
   rospy roscpp tf std_msgs sensor_msgs roslib
-  trajectory_msgs geometry_msgs nav_msgs control_msgs
+  trajectory_msgs geometry_msgs 
   message_generation
   seed_r7_description
   seed_smartactuator_sdk
@@ -30,7 +30,7 @@ endif()
 catkin_package(
   CATKIN_DEPENDS
   roscpp tf std_msgs sensor_msgs roslib
-  trajectory_msgs geometry_msgs nav_msgs control_msgs move_base_msgs
+  trajectory_msgs geometry_msgs 
   seed_r7_description
   DEPENDS
   INCLUDE_DIRS include

--- a/seed_r7_robot_interface/package.xml
+++ b/seed_r7_robot_interface/package.xml
@@ -12,18 +12,11 @@
 
   <depend>roscpp</depend>
   <depend>std_msgs</depend>
-  <depend>geometry_msgs</depend>
-  <depend>sensor_msgs</depend>
-  <depend>control_msgs</depend>
-  <depend>trajectory_msgs</depend>
   <depend>tf</depend>
-  <depend>message_generation</depend>
   <depend>roslib</depend>
-  <depend>gmapping</depend>
-  <depend>message_runtime</depend>
 
   <depend>seed_r7_description</depend>
-
+  <depend>seed_r7_typef_moveit_config</depend>
   <depend>seed_smartactuator_sdk</depend>
 
   <export/>

